api/config.js

```diff
@@ -78,6 +78,8 @@ overrides['DB_HOSTNAME'] = 'database.primary.host';
 overrides['DB_PORT'] = 'database.primary.port';
 overrides['REDIS_HOST'] = 'redis.host';
 overrides['REDIS_PORT'] = 'redis.port';
+overrides['RATELIMIT_COUNT'] = 'limit.count';
+overrides['RATELIMIT_WINDOW'] = 'limit.window';
```

Comment:
Since these are now in the overrides dictionary, the api now expects them to be required in all modes, we don't want that since ratelimiting configuration isn't really relevant when we're running our tests, so these configuration values should be removed from the `requireds` if the api isn't being run in production.